README.md

```diff
@@ -87,6 +87,7 @@ For detailed information please read the [API docs](https://pigulla.github.io/cl
 | **Map**                                      |                                                                                                             |                                                                                            |
 | `@MapContains(required, options?)`           | Checks if the given value is a Map and contains all `required` values.                                      |    [🔗](https://pigulla.github.io/class-validator-extended/functions/MapContains.html)     |
 | `@MapContainsKeys(required, options?)`       | Checks if the given value is a Map and contains all `required` keys.                                        |  [🔗](https://pigulla.github.io/class-validator-extended/functions/MapContainsKeys.html)   |
+| `@MapIncludesKeys(value, options?)`          | Checks if the given value is a Map and includes only the `value` keys, with no others.                      |  [🔗](https://pigulla.github.io/class-validator-extended/functions/MapIncludesKeys.html)   |
```

Comment:
I think that having both `MapContains...` and `MapIncludes...` might be a bit confusing as it's not at all obvious what the difference is.

I'd prefer `MapContainsKeysExactly`.